### ISSUE TITLE
Trip tags should be diversified or made to accept user tags

### ISSUE THREAD

**Issue by Greenman126** [kmichael500/planarian#24]
"sightseeing" or "inventory" seem like valid tags to me. I'm sure there are countless tags we could never think of.

People should be allowed (but discouraged) to make their own tags

**Comment by kmichael500** [comment]
@Greenman126 Should tag creation happen only in project settings or should you be able to add any tag you want when tagging something?

If you can create new tags only at the project level then I feel like it would discourage people from lazily tagging something as "Sport" instead of "Sightseeing" because they didn't scroll down the list. If it's a little harder to create tags, then I feel like there will be less of that.

We should also come up with a master list of tags so people generally don't have to create tags except in specific circumstances.

**Comment by Greenman126** [comment]
Yes. Awesome idea. Only allow it to happen at the project level.
